web/components/PostCreationModal/Form/FormSlide.tsx

```diff
@@ -0,0 +1,229 @@
+              <option value="cat">Cat</option>
+            </Select>
+          </FormControl>
+          <FormControl className="genderForm" maxW="450px">
+            <FormLabel>Gender</FormLabel>
+            <Select placeholder=" " {...register("gender")}>
+              <option value="female">Female</option>
+              <option value="male">Male</option>
+              <option value="litter">Litter</option>
+            </Select>
+          </FormControl>
+        </Stack>
+        <Stack direction={"row"} spacing={10}>
+          <FormControl className="ageForm" maxW="450px">
+            <FormLabel>Age</FormLabel>
+            <Select placeholder=" " {...register("age")}>
+              <option value="puppy">Puppy</option>
+              <option value="young">Young</option>
+              <option value="adult">Adult</option>
+              <option value="senior">Senior</option>
+              <option value="momAndPuppies">Mom & Puppies</option>
+            </Select>
+          </FormControl>
+          <FormControl className="fosterTypeForm" maxW="450px">
+            <FormLabel>Types of Foster</FormLabel>
+            <Select placeholder=" " {...register("fosterType")}>
+              <option value="return">Return</option>
+              <option value="boarding">Boarding</option>
+              <option value="temporary">Temporary</option>
+              <option value="fosterMove">Foster Move</option>
+              <option value="shelter">Shelter</option>
+              <option value="newIntake">Owner Surrender</option>
+            </Select>
+          </FormControl>
+        </Stack>
+        <FormControl className="sizeForm" maxW="450px">
+          <FormLabel>Size</FormLabel>
+          <Select placeholder=" " {...register("size")}>
+            <option value="xs">XS</option>
+            <option value="s">S</option>
+            <option value="m">M</option>
+            <option value="l">L</option>
+            <option value="xl">XL</option>
+          </Select>
+        </FormControl>
+        <FormControl className="breedForm">
+          <FormLabel>Breed</FormLabel>
+          <Select placeholder=" " {...register("breed")}>
+            <option value="americanEskimo">American Eskimo</option>
+            <option value="australianShepherd">Australian Shepherd</option>
+            <option value="...">...</option>
+            <option value="whippet">Whippet</option>
+            <option value="other">Other</option>
+          </Select>
+        </FormControl>
+      </Stack>
+      <Stack direction={"row"} spacing={10}>
+        <FormControl className="temperamentForm" maxW="450px">
+          <FormLabel>Temperament</FormLabel>
+          <Select placeholder=" " {...register("temperament")}>
+            <option value="friendly">Friendly</option>
+            <option value="scared">Scared</option>
+            <option value="active">Active</option>
+            <option value="calm">Calm</option>
+          </Select>
```

Comment:
As mentioned in #36, use React Select for all dropdowns. Some fields like this one must also be a multi-select.